dao/src/main/java/org/thingsboard/server/dao/sql/asset/AssetRepository.java

```diff
@@ -41,7 +41,8 @@ public interface AssetRepository extends JpaRepository<AssetEntity, UUID>, Expor
     AssetInfoEntity findAssetInfoById(@Param("assetId") UUID assetId);
 
     @Query("SELECT a FROM AssetEntity a WHERE a.tenantId = :tenantId " +
-            "AND (:textSearch IS NULL OR ilike(a.name, CONCAT('%', :textSearch, '%')) = true)")
+            "AND (:textSearch IS NULL OR ilike(a.name, CONCAT('%', :textSearch, '%')) = true " +
+            "  OR ilike(a.label, CONCAT('%', :textSearch, '%')) = true) ")
```

Comment:
Wrap statements related to text search in separate brackets (everywhere + don't forget about PE)